AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/Configuration/AWSPinpointAnalyticsPluginConfiguration.swift

```diff
@@ -139,22 +139,15 @@ public struct AWSPinpointAnalyticsPluginConfiguration {
             )
         }
 
-        if regionValue.isEmpty {
+        // TODO: Validate if Unknown is a valid type
+        if regionValue.isEmpty || regionValue == "Unknown" {
```

Comment:
Unknown was a valid type in [AWSCore Reference](https://aws-amplify.github.io/aws-sdk-ios/docs/reference/AWSCore/Enums/AWSRegionType.html#/c:@E@AWSRegionType@AWSRegionUnknown)

Reply:
Yep, but since that's gone we need to validate whether the `regionValue` string can still take that value